lib/src/arithmetic/floating_point/floating_point_values/floating_point_value.dart

```diff
@@ -119,135 +118,159 @@ class FloatingPointValue implements Comparable<FloatingPointValue> {
   final int _maxExp;
   final int _minExp;
 
-  /// Factory (static) constructor of a [FloatingPointValue] from
-  /// sign, mantissa and exponent
-  factory FloatingPointValue(
+  /// A Map from the (exponentWidth, mantissaWidth) pair to the
```

Comment:
I feel like we should update this doc comment to explain how it's used and how people can leverage it to override default behavior

Reply:
I am not sure of the utility of the map. We can work without it for now.  I think if people are accessing the constructors from the base class, they won't get the special behavior of the sub-class constructors that I need to add.  We need to figure out what we want here, as FPV(e=4,m=3) is not going to be the same type as FP8E4M3 since the latter does not represent infinity.

Reply:
Unfortunately the test for E4M3 calls the base class constructor and does not validate the fromDouble routine.  So we have bugs in the very basic tests for the subtypes I need to check.

Reply:
the `fromDouble` seems like a special case we can ignore for now.  i updated the docs for the mapped stuff and i think that should all work nicely for everything except the fromDouble